docs/product-highlights-and-benefits/live-context.mdx

```diff
@@ -1,65 +1,87 @@
+The Workstream Pattern Engine must be turned on to use Live Context. We’ve made this super easy to do from wherever you’re getting started.
 
-![WPE Pipeline](/wpe/pipeline.png)
+:::
 
 You can add additional context to further tailor the conversation if you’d like.
 
 ### Permissions
-If you’re a Mac user, you will need to update Pieces’ permissions in order to use workstream context. Windows users may disregard this step.
 
-<Video
-  type={'local'}
-  src={'/wpe/permissions-macos'}
-/>
+If you’re a Mac user, you will need to update Pieces’ permissions in order to use live context. Windows users may disregard this step.
+
+<Video type={'local'}  src={'/wpe/permissions-macos'}/>
 
 You can also manually update this by adding and enabling Pieces OS in the following settings:
 
 #### Privacy & Security > Accessibility
-
 ![Accessibility Permissions](/wpe/accessibility-permissions.png)
 
 #### Privacy & Security > Screen and System Audio Recording
-
 ![Screen and System Audio Recording Permissions](/wpe/screen-and-system-audio-permissions.png)
 
 ## Recommendations and Best Practices
 
-### LLM
-At this moment, we recommend using OpenAI’s cloud models as your Pieces Copilot runtime for the best experience. Due to the context length required to add workstream context, local LLMs may not allow you to have the best experience with this feature. You are of course welcome to try using local models, and other cloud models— please let us know how it goes for you!
+:::tip
+We recommend clicking around your workflow between chats, websites, emails, code, etc., while the WPE is running before asking a question to get the most value out of it (i.e., data capture).
+:::
+
+### Interaction with Language Models (LLM)
+
+- **Cloud LLM:** If you are using a cloud-based LLM, the data identified as relevant is sent to the cloud LLM for processing.
+- **Local LLM:** If you are using a local LLM, the data remains on your device, ensuring that all processing happens locally without any data leaving your device.
 
 ### Prompting
+
 As with any interaction with an LLM, good prompting practices will improve your experience greatly. Through the Workstream Pattern Engine, the Pieces Copilot is able to understand what you’re working on, including files, websites, folders, etc and with whom. Therefore, you are now able to ask even more natural questions that feel like extensions of thought. Here are some examples of what's now possible:
+
 - "Can you summarize the readme file from the pieces_for_x repo?"
 - "What did Sam have to say about the All Hands meeting in the GChat MLChat channel?"
 - "Generate a script in python using the function I saw on W3Schools to create a variable named xyz"
 - "Take the function from example_function.dart and add it as a method to the class in example_class.dart"
 - "What did Mark say about requests to the xyz api in slack?"
 
-### Use Cases and Tutorials
+## Use Cases and Tutorials
+
 1. Exception/error handling
```

Comment:
Lets add one here that's:

"5. Explaining confusing documentation"

Reply:
@mack-at-pieces Can you take another look at this, it has been updated from rosie

Reply:
Nice - I think there's still opportunity to add this one. Thoughts?

Reply:
Where would you say to add it @mack-at-pieces? In the `coding` list?

Reply:
added to research section